operator/handler/annotations.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package handler
+
+import (
+	"fmt"
+	"regexp"
+)
+
+const defaultAnnotationKeyPrefix = "tailing-sidecar.sumologic.com"
+
+var whiteSpaces = regexp.MustCompile(`\s+`)
+
+func addAnnotations(annotations map[string]string, config sidecarConfig) map[string]string {
+	if annotations == nil {
+		annotations = make(map[string]string, 0)
+	}
+
+	if config.name == "" {
+		handlerLog.Info("Missing tailing sidecar container name, cannot create per sidecar annotations",
+			"config", config)
```

Comment:
taste comment:

```suggestion
		handlerLog.Info(
			"Missing tailing sidecar container name, cannot create per sidecar annotations",
			"config", config,
		)
```

Reply:
Visual Studio Code corrects formatting to existing form... Does your IDE do formatting in different way?